.bumpversion.cfg

```diff
@@ -1,12 +1,14 @@
 [bumpversion]
-current_version = 6.0.6
+current_version = 6.0.8
 commit = true
 message = Bump Version: {current_version} → {new_version}
 tag = false
 parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)
 serialize = 
 	{major}.{minor}.{patch}
 
+[bumpversion:file(latest_version):roles/confluent.test/molecule/archive-scram-rhel/verify.yml]
```

Comment:
Can you please have a look on this PR https://github.com/confluentinc/cp-ansible/pull/1100 and get rid of hardcoded versions in the verify.yml